docs/configuration.md

```diff
@@ -37,7 +37,7 @@ directory.
 >as replacing the storage driver type with `REGISTRY_STORAGE=filesystem`, then
 >all sub-fields will be erased. As such, specifying the storage type in the
 >environment will remove all parameters related to the old storage
->configuration.
+>configuration (order *matters*).
```

Comment:
Could you elaborate on this? It's not clear to me from the context what needs to be ordered properly.

(BTW, I'll remove this paragraph in my PR that fixes environment variable handling.)